jupyter_server/services/contents/manager.py

```diff
@@ -34,6 +34,13 @@
 copy_pat = re.compile(r"\-Copy\d*\.")
 
 
+class NoWatchfilesAPI:
```

Comment:
I think `NoWatchfilesAPI` should contain the desired APIs without concrete implementations. So it can be used as a type hint for the `watchfiles` property of `ContentsManager`

Reply:
Yes so that would go in [Martin's direction](https://github.com/jupyter-server/jupyter_server/pull/783#issuecomment-1100117524).